packages/core/src/components/DataTable/index.tsx

```diff
@@ -453,13 +479,13 @@ export default withStyles(
     column_divider: {
       borderRight: ui.border,
     },
-    row: {
+    rowContainer: {
       height: '100%',
       display: 'flex',
```

Comment:
have you tried removing `flexbox` on the row container to see if it fixes the `auto` row height calculation? the `react-virtualized` examples does not have this so maybe worth trying.